VERSIONING.md

```
@@ -31,32 +31,32 @@ All commits MUST follow the Conventional Commits specification with a **mandator
 # Bad - will be rejected
-feat(browser-ui): add button component  # Sub-scope not allowed
-feat: add new feature                    # Missing scope
+feat(stagewise-ui): add button component  # Sub-scope not allowed
+feat: add new feature                     # Missing scope
 ```
 
 ### Breaking Changes
 
 Add `BREAKING CHANGE:` in the commit footer or `!` after the type:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)